internal/integrations/v4/integration/integration.go

```diff
@@ -61,6 +62,11 @@ func newDefinitionWithoutLookup(ce config2.ConfigEntry, passthroughEnv []string,
 	if err := ce.Sanitize(); err != nil {
 		return Definition{}, err
 	}
+
+	interval := getInterval(ce.Interval)
```

Comment:
what would happen here for executions that have no interval?

like command-channel requested ones, which only run once

Reply:
good question, but the that behaivor is not being changed here, I've only extract the creation of this variable outside the `Definition` struct creation below in order to have the interval value available to populate the env variable. 
My guess is that cmd channel don't use this function , otherwise the interval would already being set it to default by `getInterval()`

Reply:
I've pushed some tests for default and 0 cases